### DISCUSSION TITLE
How do i put threshold in predict yolo v8  Command Line Interface Usage

### DISCUSSION THREAD

**Discussion by faisalhazry** [ultralytics/ultralytics#2978] [Q&A]
This is predict yolo v8  Command Line Interface Usage how do i add threshold  

yolo detect predict model=best_Yolov8-seg_9.pt source='Video2_test.mp4'

**Comment by HichTala** [comment]
If you want to modify intersection over union (IoU) threshold for NMS you need to add ```iou``` flag. If you want to modify object confidence threshold it is ```conf``` flag that needs to be modify. You can for example use something like :

```
yolo detect predict model=best_Yolov8-seg_9.pt source='Video2_test.mp4' iou=0.6 conf=0.5
```

You can refer to the [documentation](https://docs.ultralytics.com/modes/predict/) to see all the configurable parameters,

Hope this help !

**Reply by glenn-jocher** [reply]
@HichTala to set a confidence threshold for predictions in YOLOv8 using the CLI, you can use the `--conf-thres` flag followed by the desired threshold value. For example, if you want to set the confidence threshold to 0.4, you would modify your command like this:

```
yolo detect predict model=best_Yolov8-seg_9.pt source='Video2_test.mp4' --conf-thres 0.4
```

This will filter out detections that have a confidence score lower than 0.4. Adjust the value according to your needs for optimal results. For more details on the available options, you can check out the Predict mode section in our documentation. Happy detecting! 😊
